uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/PublicHoliday.tpl

```diff
@@ -0,0 +1,49 @@
+{if $action eq 1 or $action eq 2 or $action eq 8}
+    {include file="CRM/HRLeaveAndAbsences/Form/PublicHoliday.tpl"}
+{else}
+    {if $rows}
+        <div id="ltype">
+            <div class="form-item">
+                {strip}
+                    {* handle enable/disable actions*}
+                    {include file="CRM/common/enableDisableApi.tpl"}
+                    <table cellpadding="0" cellspacing="0" border="0" class="hrleaveandabsences-entity-list">
+                        <thead class="sticky">
+                            <th>{ts}Title{/ts}</th>
+                            <th>{ts}Date{/ts}</th>
+                            <th>{ts}Enabled/Disabled{/ts}</th>
+                            <th></th>
+                        </thead>
+                        {foreach from=$rows item=row}
+                            <tr id="PublicHoliday-{$row.id}" class="crm-entity {cycle values="odd-row,even-row"} {$row.class}{if NOT $row.is_active} disabled{/if}">
+                                <td data-field="title">{$row.title}</td>
+                                <td>{$row.date}</td>
```

Comment:
Please change this to:
`<td>{$row.date|crmDate}</td>`
To display the date using the format defined in the system settings